docs/sources/flow/reference/components/otelcol.exporter.loadbalancing.md

```diff
@@ -264,6 +267,90 @@ Name | Type | Description
 * logs
 * traces
 
+## Choosing a load balancing strategy
+
+<!-- TODO: Mention gropubytrace processor when Flow supports it -->
+<!-- TODO: Should we run more than 1 LB instance for better resiliency and spreading out the load? -->
+
+There are various stateful Flow components which require specific load balancing strategies.
+<!-- TODO: "stateful" should be a link to the other doc which defines it -->
+
+### otelcol.processor.tail_sampling
+<!-- TODO: Add a picture of the architecture?  -->
+All spans for a given trace ID must go to the same tail sampling Agent.
+* This can be done by configuring `otelcol.exporter.loadbalancing` with `routing_key = "traceID"`.
+* If this is not done, the sampling decision may be incorrect. This is because the tail sampler must
+  have a full view of the trace when making a sampling decision.
+  * For example, a `rate_limiting` tail sampling strategy may incorrectly pass through 
+    more spans than expected if the spans for the same trace are spread out to more than one Agent instance.
+
+<!-- Make "rate limiting" a URL to the tail sampler doc -->
+
+### otelcol.connector.spanmetrics
+All spans for a given `service.name` must go to the same spanmetrics Agent.
+* This can be done by configuring `otelcol.exporter.loadbalancing` with `routing_key = "service"`.
+* If this is not done, metrics generated from spans might be incorrect.
+  * For example, if similar spans for the same `service.name` end up on different Agent instances,
+    the two Agents will have identical metric series for calculating span latency, errors, and number of requests.
+  * When both Agents attempt to write the metrics to a database such as Mimir, the series may clash with each other.
+  * At best this will lead to an error in the Agent and a rejected write to the metrics database. 
+  * At worst, it could lead to an inaccurate data due to overlapping samples for the metric series.
```

Comment:
This is a very dense and hard to understand subject. The reason I wrote them as bullets is that I felt it divides this information in bullets makes it easier to digest than having one big paragraph. But if you think this makes more sense from a writing point of view, I'm happy to change it.

Reply:
Yeah I agree, it's a complex topic. The problem with the bullet points is that they should be "in agreement" or to put it another way, they should all follow the same flow.... which these do not. Instead of one big paragraph, try out the double LF which spaces out the sentences vs clumping it all int one big lump.

Either way will work... most important to get the info published without being overly pedantic on my side :-)